tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -0,0 +1,81 @@
+"""Schema for HEADER row of all submission types."""
+
+
+from ...util import SchemaManager
+from ...fields import EncryptedField, Field, tanf_ssn_decryption_func
+from ...row_schema import RowSchema
+from ... import validators
+from tdpservice.search_indexes.models.tanf import TANF_T5
+
+
+t5 = SchemaManager(
+      schemas=[
+        RowSchema(
+          model=TANF_T5,
+          preparsing_validators=[
+              validators.hasLength(71),
+          ],
+          postparsing_validators=[],
+          fields=[
+              Field(item=1, name='RecordType', type='string', startIndex=0, endIndex=2,
+                    required=True, validators=[]),
+              Field(item=4, name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
+                    required=True, validators=[]),
+              Field(item=6, name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
+                    required=True, validators=[]),
+              Field(item=14, name='FAMILY_AFFILIATION', type='number', startIndex=19, endIndex=20,
+                    required=True, validators=[]),
+              Field(item=15, name='DATE_OF_BIRTH', type='number', startIndex=20, endIndex=28,
+                    required=True, validators=[]),
+              EncryptedField(decryption_func=tanf_ssn_decryption_func, item=16, name='SSN', type='string',
+                             startIndex=28, endIndex=37, required=True, validators=[]),
+              Field(item=17, name='RACE_HISPANIC', type='string', startIndex=37, endIndex=38,
```

Comment:
#2384 adds the sub-item for `RACE_x` fields to the item numbers (`33A`, `33B` etc.) - should we do the same here?

Reply:
same for item 19 below